### DISCUSSION TITLE
Welcome to databricks-grafana Discussions!

### DISCUSSION THREAD

**Discussion by mullerpeter** [mullerpeter/databricks-grafana#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by epa095** [comment]
Awesome that you started this project! Thanks!

**Comment by rajkve5055** [comment]
Hi @mullerpeter,

Curious to understand if the connector will support an autorefresh of the charts in Grafana (with continuous polling of data from databricks delta table)

**Reply by mullerpeter** [reply]
Yes it will support the normal auto refresh like other data source plugins

**Comment by Dammi87** [comment]
Dude, awesome work. The fact that they want to keep databricks interface behind an insanely expensive paywall is super annoying.
